spec/jekyll-feed_spec.rb

```diff
@@ -125,9 +130,9 @@
 
     it "doesn't include the item's excerpt if blank" do
       if Gem::Version.new(Jekyll::VERSION) > Gem::Version.new('3')
-        post = feed.items.first
```

Comment:
This is a very fragile test. Will need to be updated anytime a new post is added to the feed or an old post is updated, since posts are sorted by updated date. :unamused: